FirebaseTestProject/Models/Post.swift

```diff
@@ -23,6 +23,39 @@ struct Post: FirebaseConvertable {
         self.timestamp = Date()
     }
     static let testPost = Post(title: "Title", text: "Content", author: "First Last")
+    
+    func contains(_ str:String) -> Bool {
```

Comment:
This is admittedly nitpicking, but I would put a space after the colon (so it would be `func contains(_ str: String) -> Bool {`)